src/shared/components/search-result/search-result.component.ts

```diff
@@ -6,12 +6,13 @@ import { CaseField, CaseState, CaseType, CaseView, DisplayMode,
   DRAFT_PREFIX, Jurisdiction, PaginationMetadata, SearchResultView, SearchResultViewColumn,
   SearchResultViewItem, SearchResultViewItemComparator, SortOrder, SortParameters } from '../../domain';
 import { CaseReferencePipe } from '../../pipes';
-import { ActivityService, SearchResultViewItemComparatorFactory } from '../../services';
+import { ActivityService, SearchResultViewItemComparatorFactory, BrowserService } from '../../services';
 
 @Component({
   selector: 'ccd-search-result',
   templateUrl: './search-result.component.html',
-  styleUrls: ['./search-result.component.scss']
+  styleUrls: ['./search-result.component.scss'],
+  providers: [BrowserService]
```

Comment:
can this providers move to search-result.module.ts?

Reply:
done